.ci/setup.sh

```diff
@@ -4,30 +4,17 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 #
-
 set -e
 
 cidir=$(dirname "$0")
 source "${cidir}/lib.sh"
 
-#Note: If add clearlinux as supported CI use a stateless os-release file
-source /etc/os-release
-
-if [ "$ID" == fedora ];then
-	sudo -E dnf -y install automake yamllint coreutils moreutils bc make gcc
-elif [ "$ID" == centos ];then
-	sudo -E yum -y install epel-release
-	sudo -E yum -y install automake yamllint coreutils moreutils bc
-elif [ "$ID" == ubuntu ];then
-	sudo apt-get -qq update
-	sudo apt-get install -y -qq make automake qemu-utils python-pip coreutils moreutils bc
-	sudo pip install yamllint
-else
-	echo "Linux distribution not supported"
-fi
+clone_tests_repo
 
+pushd "${tests_repo_dir}"
+.ci/setup.sh
```

Comment:
`.ci/setup.sh` already runs on https://github.com/kata-containers/osbuilder/blob/f17b5ee8f8531bd56abd8baded5d98431933ed86/tests/test_images.sh#L278

so with this, we are running the whole setup twice.

Reply:
oops, I totally missed this comment. Thank you for the heads up @chavafg